### DISCUSSION TITLE
Hiding old entires

### DISCUSSION THREAD

**Discussion by Jorenar** [Athou/commafeed#1029] [Q&A] [answered]
I have a few thousand (old) news just from fresh import of OPML file. Is there any value I can set in H2 database that will make them not appear at all (even as read), but still be recognized when fetching updates?

**Comment by Athou** [comment, marked as answer]
I'm afraid not. There's the `app.maxFeedCapacity` setting that allows you to restrict the number of entries to keep per feed but it's currently a global setting for all feeds.

May I ask why you'd like to avoid seeing old entries, even if they're marked as read?

**Reply by Jorenar** [reply]
Now that I think about it, perhaps I don't really need it? I guess I can just reverse order to descending when browsing read articles.

In QuiteRSS (which I want to migrate from) there is an option "Clean Up..." which marks entries as deleted in database (and from what I see, also removes no longer need data like e.g. articles' content) and doesn't show them up anymore on the list.
I wanted to replicate that, even if by manually running queries in H2 Console.
